lib/rubocop/cop/rails/active_record_bulk_persistence.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # Checks for iterative save/create/update methods instead of using insert_all/upsert_all
+      #
+      # @example
+      #   # bad
+      #   events.each do |user|
+      #     event.save!
+      #   end
+      #
+      #   # good
```

Comment:
I'm a little wary of calling this 'good' without a warning since it will skip validations and callbacks, and that may not always be desirable.

Reply:
I added a `@safety` section warning about the validations/callbacks.